boa_engine/src/object/builtins/jsarraybuffer.rs

```diff
@@ -92,13 +123,70 @@ impl JsArrayBuffer {
     }
 
     /// Returns the byte length of the array buffer.
+    ///
+    /// ```
+    /// # use boa_engine::{
+    /// # object::builtins::JsArrayBuffer,
+    /// # Context,
+    /// # };
+    /// # // Initialize context
+    /// # let context = &mut Context::default();
+    /// // Create a buffer from a chunk of data
+    /// let data_block: Vec<u8> = (0..5).collect();
+    /// let array_buffer = JsArrayBuffer::from_byte_block(data_block, context).unwrap();
+    ///
+    /// // Take the inner buffer
+    /// let buffer_length = array_buffer.byte_length(context);
+    ///
+    /// assert_eq!(buffer_length, 5_usize);
```

Comment:
```suggestion
    /// assert_eq!(buffer_length, 5);
```
Can't the compiler infer the type here?

Reply:
Yeah, I think your right. I was being overly safe when I typed that out since writing and testing the docs can be a tad tedious at times.